manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -18,30 +42,23 @@ rules:
     resources: ["validatingwebhookconfigurations"]
     verbs: ["get", "list", "watch", "update"]
 
-  # permissions to verify the webhook is ready and rejecting
-  # invalid config. We use --server-dry-run so no config is persisted.
-  - apiGroups: ["networking.istio.io"]
-    verbs: ["create"]
-    resources: ["gateways"]
-
   # istio configuration
   - apiGroups: ["config.istio.io", "security.istio.io", "networking.istio.io", "authentication.istio.io"]
-{{- if .Values.global.istiod.enableAnalysis }}
-    verbs: ["get", "watch", "list", "update"]
-{{- else }}
     verbs: ["get", "watch", "list"]
-{{- end }}
     resources: ["*"]
+{{- if .Values.global.istiod.enableAnalysis }}
```

Comment:
Let's just add update to the line above. It doesn't make sense to use a global flag just for this. 

Almost nothing else in base is custom/global based - it is nice to have a plain yaml.